tests/fillers/vm_fillers/paygas_fillers.py

```diff
@@ -32,15 +25,18 @@
     1: 1,
 })
 
-broke_caller = (caller, "balance", 0)
-solvent_caller = (caller, "balance", 1000000)
+paygas_successful = (address, "storage", 2, 2)  # return code + 1
+paygas_failed = (address, "storage", 2, 1)  # return code + 1
```

Comment:
In `paygas_in_call_test` test case, there doesn't seem to be an `ADD` operation.

Reply:
It is, but it's hidden in the `paygas_contract`. See https://github.com/jannikluhn/py-evm/blob/0812496bab24c91a9b9b1e12bdb814a080958809/tests/fillers/vm_fillers/paygas_fillers.py#L37

Reply:
But aren't both `paygas_successful` and `paygas_failed` the cases for `address`(which is generated by `generate_random_address()`)?
`paygas_successful ` is the case for `address` but shouldn't `paygas_failed` be the case for `paygas_contract_address`?

Reply:
Ah, now I get what you mean. This is what's going on: The paygas contract returns the return code (+ 1), but the "main" contract (at `address`) stores it in its storage. So to check for success/failure, one has to check the storage at `address`, even if the actual call was done at `paygas_contract_address`. See https://github.com/jannikluhn/py-evm/blob/0812496bab24c91a9b9b1e12bdb814a080958809/tests/fillers/vm_fillers/paygas_fillers.py#L274

Reply:
Ah I see. I didn't see that in `paygas_contract`, result of `["ADD", 1, ["PAYGAS", 1]]` is store in memory and then returned.